worker-languagedetection/README.md

```diff
@@ -24,8 +30,20 @@ Worker configuration is supported through the following environment variables:
     Default: `worker-out`  
     Sets the output queue where results are returned
 
+### Per Tenant Settings
+
+When `fieldSpecs` is provided to the worker via `customData` the worker's behaviour will change and the worker will begin [Multi-Field Processing](#multi-field-processing).
+
+ - `fieldSpecs`  
+    This field specifies to the worker on what fields to perform language detection. It is a string of comma-seperated values, each value will either be a complete field name or a partial field name supplied with a wildcard character "*".
+    
+**Example:** `{"fieldSpecs": "CONTENT_*, TITLE, SUBJECT"}`
```

Comment:
Split the example onto a separate line, and the description of the example onto another line below that.